lib/airbrake-ruby.rb

```diff
@@ -84,6 +85,10 @@ module Airbrake
   #  special cases where we need to work around older implementations
   JRUBY = (RUBY_ENGINE == 'jruby')
 
+  # @return [Boolean] true if current Ruby is MRI < 2.7. The result is used
+  #  to guard against using deprecated features which will be removed in MRI 3.
+  HAS_SAFE_LEVEL = (RUBY_ENGINE == 'ruby' && RUBY_VERSION < '2.7')
```

Comment:
We use `Gem::Version.new(RUBY_VERSION)` for comparisons. This is because it can handle corner cases that simple string comparison cannot.